hc/accounts/tests/test_profile.py

```diff
@@ -108,3 +116,27 @@ def test_it_shows_badges(self):
         self.assertNotContains(r, "bobs-tag.svg")
 
     ### Test it creates and revokes API key
+    def test_it_creates_api_key(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        self.client.get("/accounts/profile/")
+
+        # Set the API key, in this case "abc"
+        self.alice.profile.refresh_from_db()
+        api_key = self.alice.profile.api_key
+
+        # Assert that the API key is not None
+        self.assertIsNotNone(api_key)
+
+    def test_it_revokes_api_key(self):
+        self.client.login(username="alice@example.org", password="password")
+        
+        form = {"revoke_api_key": "dfg"}
+        self.client.post("/accounts/profile/", form)
+
+        # API key should be set to None
+        self.alice.profile.refresh_from_db()
+        api_key = self.alice.profile.api_key
+
+        # Assert that the API key is None
+        self.assertEqual(api_key, "")
```

Comment:
add assertions for the status codes as well for the response of the post request

Reply:
and there should be an assertion that test a response `"The API key has been revoked!"` is present as well.